CONTRIBUTING.md

```diff
@@ -4,28 +4,34 @@
 
 To run this project, you will need:
 
-- Node.js >= v7.10.0, use nvm - [install instructions](https://github.com/creationix/nvm#install-script)
-- Yarn >= v0.23.4 - [install instructions](https://yarnpkg.com/en/docs/install#alternatives-tab)
+- Node.js >= v8.10.0, use nvm - [install instructions](https://github.com/creationix/nvm#install-script)
```

Comment:
Maybe use the latest LTS Node and Yarn now?

Reply:
I synced those values with the `engines` property of the `package.json`. We can upgrade them after the merge of those PRs.